packages/core/lib/commands/init/initSource/truffle-config.js

```diff
@@ -84,7 +87,7 @@ module.exports = {
   // Configure your compilers
   compilers: {
     solc: {
-      version: "0.8.15",      // Fetch exact version from solc-bin (default: truffle's version)
+      version: "0.8.15", // Fetch exact version from solc-bin (default: truffle's version)
```

Comment:
It looks like this from [here](https://github.com/trufflesuite/truffle/pull/5320#discussion_r928075653) isn't actually resolved.

Reply:
The whitespace.

Reply:
Yeah! I guess I missed that. Will do the required changes. Thanks @cliffoo.